### DISCUSSION TITLE
open foocus

### DISCUSSION THREAD

**Discussion by rika-maker** [lllyasviel/Fooocus#3371] [Q&A] [answered]
i really need help i installed fooocus correctly and then after installing it press any key to continue but doesn't open fooocus automatically  ...so i searched on discussions and found that i can open it from http://127.0.0.1:7865/ and didn't but doesn't open show me this 
![image](https://github.com/user-attachments/assets/3ab8e32b-672f-4ccc-aa4e-cd1c8e29f5b0)
 so i checked the connection it's working, don't have a proxy and i turned off the Windows firewall and restarted still..found again that maybe from not enough memory need at least 40GB and i have 122 on my Windows default automated system swap, i have Nvidia GeForce version 425.31...I try to solve every problem last 3 days and I don't know what els to do now ..any help please

**Comment by mashb1t** [comment, marked as answer]
Driver 425.31 is very old (from April 2019), please upgrade.
Also make sure you've followed the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) after checking the terminal outputs, where you 99% will find an error providing you more information about what has happened.

**Reply by rika-maker** [reply]
This after I updated ... I have two drivers intel and Nividia that I could update is Nividia I don't know if I can download the newer one but I will try ..thank you for the replay... I see the guide the thing Is I don't understand those thing so I try every app and see what happens or ask for help from AI chat and follow the steps
